config/checkstyle-input-suppressions.xml

```diff
@@ -70,11 +70,7 @@
   <suppress id="noCheckstyleInInputs"
           files="[\\/]checks[\\/]metrics[\\/]classdataabstractioncoupling[\\/]InputClassDataAbstractionCouplingExcludedPackagesDirectPackages.java"/>
   <suppress id="noCheckstyleInInputs"
-          files="[\\/]checks[\\/]metrics[\\/]classfanoutcomplexity[\\/]InputClassFanOutComplexityExcludedPackagesAllIgnored.java"/>
-  <suppress id="noCheckstyleInInputs"
-          files="[\\/]checks[\\/]metrics[\\/]classfanoutcomplexity[\\/]InputClassFanOutComplexityExcludedPackagesCommonPackage.java"/>
-  <suppress id="noCheckstyleInInputs"
-          files="[\\/]checks[\\/]metrics[\\/]classfanoutcomplexity[\\/]InputClassFanOutComplexityExcludedPackagesDirectPackages.java"/>
+          files="[\\/]checks[\\/]whitespace[\\/]emptylineseparator[\\/]InputEmptyLineSeparator2.java"/>
```

Comment:
@romani I am inclined to allow these to stay as permanent suppressions, since it looks like we need specific imports for this check and all imports are of classes that live in this file. Thoughts?

Reply:
Yes

Reply:
@romani I think we have a misunderstanding here, I am proposing that we allow all suppressions for this check to be permanent since check logic relies heavily on imports. We are losing test cases with the current changes.

Reply:
yes, allow suppression and revert all changes to Inputs

Reply:
@Rohanraj123 to be clear: the only changes expected in this PR are to move suppressions to permanent suppression area in the same file with a comment.